### PR TITLE
Replace SNP_VERSION & SNP_DB with PRIMER_TEXT

### PR DESCRIPTION
**Pull request by JasonLing95**
# Changes
- Replace `SNP_VERSION` and `SNP_DB` with `PRIMER_TEXT`

---
This change is [Reviewable](https://reviewable.io/reviews/eastgenomics/primer_designer/5)

### CODE REVIEW

File: bin/primer_designer_region.py [hunk 1]
```diff
@@ -1319,12 +1327,14 @@ def pretty_pdf_fusion_mappings(
                 target_sequence = coord_dict[0]['SEQ']
                 tagged_string = coord_dict[0]['TSEQ']
 
-                if coord_dict[0]['SIDE'] == "<" and coord_dict[0]['STRAND'] == "-1":
+                if coord_dict[0]['SIDE'] == "<" and \
```
Comment (jethror1) [inline bin/primer_designer_region.py]: I think this is still against pep8 as the following line is wrongly indented, this is more correct I think:

```
if (coord_dict[0]['SIDE'] == "<" and
    coord_dict[0]['STRAND'] == "-1"
    ):
    base1 = int(coord_dict[0]['POS']) + FLANK
```

https://peps.python.org/pep-0008/#indentation

File: bin/primer_designer_region.py [hunk 2]
```diff
@@ -184,17 +184,25 @@ def flip_fusion_seq(self, seqs_dict):
                 seqs_dict[i]['TSEQ'] = seqs_dict[i]['TSEQ'][::-1]
                 seqs_dict[i]['DARK_SIDE'] = ">"
 
-        if seqs_dict[0]['DARK_SIDE'] == ">" and seqs_dict[1]['DARK_SIDE'] == "<":
+        if seqs_dict[0]['DARK_SIDE'] == ">" and \
+                seqs_dict[1]['DARK_SIDE'] == "<":
             # one set seqs before and after breakpoint => valid (?)
-            target_sequence = ''.join([seqs_dict[0]['SEQ'], seqs_dict[1]['SEQ']])
```
Comment (jethror1) [inline bin/primer_designer_region.py:189]: why is this being deleted?

Comment (JasonLing95) [reply bin/primer_designer_region.py:189]: it's not deleted

File: bin/primer_designer_region.py (hunk 1 above)
Comment (JasonLing95) [reply bin/primer_designer_region.py]: changed all to use bracket rather than \

**Review by jethror1 — APPROVED** [review]
> Reviewed 3 of 4 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/primer_designer/5)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @JasonLing95)

**Pull request merged**